test/inductor/test_fused_attention.py

```diff
@@ -315,6 +317,7 @@ def dot_prod_attention(
             checkpoint_wrapper(dot_prod_attention), contains=False, has_dropout=True
         )
 
+    @skipIfRocm  # AssertionError: expected size 4==4, stride 32==64 at dim=0
```

Comment:
Is this going to cause issues for people using compile + rocm?

Reply:
We fixed this in commit https://github.com/pytorch/pytorch/pull/124885/commits/0ff88e2109ec2a4a2f2b5ced54c84b3a2dee9596, by forcing the rewriter to use `aten._scaled_dot_product_attention_math` on ROCM